inc/thorvg.h

```diff
@@ -1021,6 +1034,17 @@ class TVG_EXPORT Shape final : public Paint
      */
     FillRule fillRule() const noexcept;
 
+    /**
```

Comment:
I think the requirement is close to 0. Let's drop getter. I think it's not necessary.